packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -51,6 +51,7 @@ export function getInspectorModeAttributes(
     environment:
       element.getAttribute(InspectorModeDataAttributes.ENVIRONMENT) ?? fallbackProps.environment,
     space: element.getAttribute(InspectorModeDataAttributes.SPACE) ?? fallbackProps.space,
+    manuallyTagged: element.hasAttribute(InspectorModeDataAttributes.MANUALLY_TAGGED),
```

Comment:
Since we convert the encoded strings to normal data attributes but without the new manually tagged flag and in [line 246 below](https://github.com/contentful/live-preview/pull/725/files#diff-7371daf98a723e8f1567e59dd41ac1acfe948cfc488c6cbf4ede34b6e4213021R246-R248) you are only checking that the new attribute doesn't exist. 

Wouldn't this report auto-tagged as manually tagged if called twice on the dom? I know we probably don't care about its value anymore but still feels wrong.

Maybe doing it the other way around would be better, like adding `data-contentful-auto-tagged=true` or `data-contentful-tagging="auto"`.

Reply:
I did it the other way around first and @chrishelgert suggested the opposite. I originally had the data attribute as auto and changed it to manual. This was to align with to how vercel does it: https://github.com/contentful/live-preview/pull/725#discussion_r1599702366

Reply:
> Wouldn't this report auto-tagged as manually tagged if called twice on the dom? I know we probably don't care about its value anymore but still feels wrong.

I would say manual tagging should take priority over autotagging

Reply:
Additional thought: 
Longterm we will have more auto-tagged than manual tagged fields (🤞 )
So adding it on manual fields should lead to a smaller fingerprint, than on auto-tagged fields

Reply:
@z0al 
> To fix the current way I think we need to
As I see it we have 2 states, manually tagged or auto tagged so if if it has the attribute then it's manually tagged if it doesn't then it's auto tagged. How would your suggestion fix what is currently here?

Reply:
Also agree with @chrishelgert here that just tagging the manual has a smaller footprint